src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public static final int FIGURE_TYPES = 5;
+    public static final int SIDE_MAX = 10;
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
```suggestion
    public static final int SIDE_MAX = 10;
    private ColorSupplier colorSupplier = new ColorSupplier();

    public Figure getRandomFigure() {
```
краще перенести це поле з методу до класу, так як нам не потрібно для кожного виклику методу створювати новий об'єкт colorSupplier